### ISSUE TITLE
SDists produces warnings with twine check

### ISSUE THREAD

**Issue by henryiii** [takluyver/flit#414]
The SDists produce warnings like this:

```console
$ pipx run twine check dist/cookie_flit621-0.0.1.tar.gz
Checking dist/cookie_flit621-0.0.1.tar.gz: PASSED, with warnings
  warning: `long_description_content_type` missing. defaulting to `text/x-rst`.
  warning: `long_description` missing.
```

I believe this would be fixed by #410.

Looking inside PKG-INFO, I see #409's issue (not exactly mangled, but has empty fields that aren't needed in newer metadata versions):

```
Metadata-Version: 1.1
Name: cookie_flit621
Version: 0.0.1
Summary: A great package.
Home-page: None
Author: None
Author-email: My Name <me@email.com>
```

The METADATA file in the wheel is fine, so I think the best fix is #410.

You can recreate this with https://github.com/scikit-hep/cookie/pull/21

**Comment by henryiii** [comment]
Classic configuration SDists fail the check too. Run from inside scikit-hep/cookie:

```console
$ pipx run nox -s "dist(flit)"
```

or, from anywhere

```console
$ pipx run cookiecutter gh:scikit-hep/cookie
$ # Chose flit when asked
$ cd <package directory>
$ pipx run build --sdist
$ pipx run twine dist/*
```

**Comment by takluyver** [comment]
Thanks @henryiii . Closing as I understand this is fixed by #410. We can reopen this - or just make another issue - if it turns out that wasn't the whole story.
